modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,297 @@
+
+| `table_use_colgroups`         | `false`                     | `true`    | Changed as part of modernising {productname}’s default behavior.      | see xref:changed-plugins-table[Table]
+|===
+
+// end::default-value-changes[]
+
+// tag::previously-deprecated-items-now-removed[]
+[[previously-deprecated-items-now-removed]]
+== Previously deprecated items now removed
+
+The following elements were previously deprecated and have, with this release, been removed entirely from {productname}.
+
+[cols="1,1,1"]
+|===
+| Item                                  | Element       | Notes
+
+| `$`                                   | API           | `$` was a shorthand function of `DomQuery` and `Sizzle`.
+
+| `addComponents`                       | API           | From `AddOnManager`.
+
+| `autoresize_on_init`                  | Option        |
+
+| `block_elements`                      | Schema option |
+
+| `boolean_attributes`                  | Schema option |
+
+| `Class`                               | API           |
+
+| `clearInterval`                       | API           | From `Delay`.
+
+| `clearTimeout`                        | API           | From `Delay`.
+
+| `Color`                               | API           |
+
+| `content_editable_state`              | Option        |
+
+| `debounce`                            | API           | From `Delay`.
+
+| `dependencies`                        | API           | From `AddOnManager`.
+
+| `DomQuery`                            | API           |
+
+| `editor_deselector`                   | Option        | From `EditorManager`.
+
+| `editors`                             | Property      | From `EditorManager`.
+
+| `editor_selector`                     | Option        | From `EditorManager`.
+
+| `elements`                            | Option        | From `EditorManager`.
+
+| `execCallback`                        | API           |
+
+| `file_browser_callback_types`         | Option        |
+
+| `filepicker_validator_handler`        | Option        | Superseded by `file_picker_validator_handler`.
+
+| `force_hex_style_colors`              | Option        |
+
+| `force_p_newlines`                    | Option        | Superseded by `forced_root_block`.
+
+| `forced_root_black: false`            | Option        | `forced_root_black` must be a non-empty string and cannot take a value of `false`.
+
+| `gecko_spellcheck`                    | Option        | Replaced by `browser_spellcheck`.
+
+| `images_dataimg_filter`               | Option        |
+
+| `JSON`                                | API           |
+
+| `JSONP`                               | API           |
+
+| `JSONRequest`                         | API           |
+
+| `mode`                                | Option        | From `EditorManager`.
+
+| `move_caret_before_on_enter_elements` | Schema option |
```

Comment:
This and the other schema options we undocumented previously. Are we sure we want to call attention to them here? @metricjs would be keen to get your thoughts on this.

At the very least they should probably be in the section/table at the bottom instead.

Reply:
Good point. I think the table at the end of this page makes sense